lib/ziti-tunnel/ziti_tunnel.c

```diff
@@ -96,7 +96,7 @@ tunneler_context ziti_tunneler_init(tunneler_sdk_options *opts, uv_loop_t *loop)
 
 void ziti_tunnel_commit_routes(tunneler_context tnlr_ctx) {
     if (tnlr_ctx->opts.netif_driver == NULL) {
-        TNL_LOG(WARN, "No netif_driver found tun is running in host only mode and intercepts are disabled");
+        TNL_LOG(DEBUG, "No netif_driver found tun is running in host only mode and intercepts are disabled");
```

Comment:
I think this is fine and I'm approving, but for the record:

I don't like the original logic here that assumes host-only mode because the netif_driver is null. In a perfect world I'd think the level should be `DEBUG` if host-only and `ERROR` otherwise. Maybe the run mode should be stored in the tunneler context too, so decisions like this are easy to make outside of ziti-edge-tunnel.c

While we're touching this line could the run-on sentence be fixed up?